frontend-react/src/components/StaticCompare.tsx

```diff
@@ -0,0 +1,143 @@
+import { ReactElement, useCallback, useEffect, useState } from "react";
+import DOMPurify from "dompurify";
+import { ScrollSync, ScrollSyncPane } from "react-scroll-sync";
+
+import { Diff, SES_TYPE } from "../utils/diff";
```

Comment:
out of scope since this is not a new thing, but why is `Diff` in caps here?

Reply:
This was a library I ported to typescript. It may be historical. I still need to contribute back the port to the source project.

We _really_ need our linter to enforce cap conventions.

Reply:
lol one can dream